src/MessageHandler.py

```diff
@@ -0,0 +1,40 @@
+import urllib2
+from src.CastsCommandHandler import CastsCommandHandler
+from src.MessageRouter import MessageRouter
+from src.StreamsCommandHandler import StreamsCommandHandler
+
+
+class MessageHandler:
```

Comment:
I could stuff this code all into fafbot.py yes, though rather keep it nicely separated. One very concrete difference this creates ATM is that the tests can run without having PySide installed, as per #23.

You are right about the name, I'm also not happy about it. Will try kicking this into a bit better form.